CMakeLists.txt

```diff
@@ -46,6 +47,23 @@ add_subdirectory(demos)
 add_subdirectory(benchs)
 add_subdirectory(tutorial/cpp)
 
+if (FAISS_ENABLE_JEMALLOC)
+  # Use jemalloc for memory allocation if FAISS_ENABLE_JEMALLOC is set.
+  # In case faiss is a sub-project of another project,
```

Comment:
For extra clarity, this comment should say "In case faiss is a sub-project of another project that uses jemalloc, ..."